<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinInsercaoMissao</class>
 <widget class="QDialog" name="WinInsercaoMissao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserção de Missões</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="Cancelar">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Ok">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>411</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Identificação da missão</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditIdMissao"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Tipo de Missão</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditTipoMissao"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="idade">
        <property name="text">
         <string>Objetivo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditObjetivo"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="datadenascimento">
        <property name="text">
         <string>Brigada Responsavel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditBrigadaResponsavel"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
